macros/validation/validate_expression_metrics.sql

```diff
@@ -0,0 +1,13 @@
+{% macro validate_expression_metrics(full_set) %}
```

Comment:
```suggestion
{% macro validate_expression_metrics(metric_names) %}
```

In standalone macros, I'd suggest using variable names that make sense on their own, instead of carrying through the variable name from the original calling place. 

This is particularly relevant for macros that are pulled out for DRYness purposes, but is a reasonable strategy across the board which is why I suggest it here.

Reply:
That makes sense to me. Full set makes sense in the context of the `get_metric_sql` model but your suggested name makes more sense for this macro.